auth-service/src/main/kotlin/uk/gov/communities/delta/auth/plugins/StatusPages.kt

```diff
@@ -52,6 +52,21 @@ fun Application.configureStatusPages(deltaWebsiteUrl: String, ssoConfig: AzureAD
                     logger.error("Failed to render Delta set password form page after rate limit", e)
                     call.respondText("Failed to render set password form page after reaching rate limit. Request id ${call.callId}")
                 }
+            else if (call.request.path().contains("/reset-password"))
+                try {
+                    call.respond(
+                        ThymeleafContent(
+                            "password-form",
+                            mapOf(
+                                "deltaUrl" to deltaConfig.deltaWebsiteUrl,
+                                "message" to tooManyRequestsErrorMessage,
+                            )
+                        )
+                    )
```

Comment:
At some point we should move these to methods on the controllers rather than this also needing to know about the templates. Let's do that another time though

Reply:
Raised DT-618 :)